app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryEvent.kt

```diff
@@ -15,7 +16,7 @@ data class EditFrequencyClicked(val frequency: DrugFrequency) : CustomDrugEntryE
 
 data class FrequencyEdited(val frequency: DrugFrequency) : CustomDrugEntryEvent()
 
-object AddMedicineButtonClicked : CustomDrugEntryEvent()
+data class AddMedicineButtonClicked(val patientUuid: UUID) : CustomDrugEntryEvent()
```

Comment:
Can't we just have the `patientUuid` in the model instead of passing for every event that requires it? 🤔

Reply:
Umm.. We only needed it when the save button is clicked and the `patientUuid` parameter is present in the screen key which is why I didn't see a reason to add it.

Reply:
Ah, makes sense if this is the only place 👍🏾